docs/src/ordered_containers.md

```diff
@@ -44,3 +44,9 @@ end
 The insertion order is conserved when iterating on the dictionary itself,
 its keys (through `keys(d)`), or its values (through `values(d)`).
 All standard `Associative` and `Dict` functions are available for `OrderedDicts`
+
+# LittleDict
+```@docs
+LittleDict
+freeze
```

Comment:
I guess it would be nice to have a simple example here.  It would be largely the same--maybe identical--to the `OrderedDict` example, but if someone is looking at the docs, it would be nice to see.

Reply:
done